backend/src/main/java/com/google/blackswan/servlets/CronServlet.java

```diff
@@ -38,19 +42,52 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // TODO: Update or clean up log message. 
     log.info("Cron job ran.");
-    
-    // TODO: Logic for cron job to run blackswan mock. (#13)
-    storeAlertInDatastore();
-    
+
+    // Simple logic for cron job, since we only have one set of alerts for now. 
+    clearCurrentAlertsInDatastore(); // Need to clear alerts as we only have one set of alerts now. 
+    storeAlertsInDatastoreSimple();
     response.setStatus(HttpServletResponse.SC_ACCEPTED); 
   }
 
-  /** Temporary method for storing dummy alert into datastore. */
-  private void storeAlertInDatastore() {
-    AlertGenerator testAlertGenerator = new DummyAlertGenerator(new DummyAnomalyGenerator());
-    List<Alert> alerts = testAlertGenerator.getAlerts();
+  private void clearCurrentAlertsInDatastore() {
+    Query query = new Query(Alert.ALERT_ENTITY_KIND);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    // Iterate through the entities to delete all comment objects.
```

Comment:
why are they `comment` objects?

Reply:
oops, i forgot to change the comments. i was basing this off code i wrote before...